action.yml

```diff
@@ -21,6 +21,16 @@ inputs:
     required: false
     type: boolean
     default: false
+  warnings-as-errors:
+    description: 'If set to true then any errors produced whilst loading the package will be treated as errors'
+    required: false
+    type: boolean
+    default: false
```

Comment:
I wasn't sure whether to have the default as `true` or `false`. To be more consistent with the old behaviour, it should be `false`, but to me it feels better to aim to have no warnings when a package loads. This, however, may introduce a lot of work if there are lots of places with warnings.

Reply:
For now it needs to be `false` as otherwise we break backwards compatibility.

However we could change the default to `true` in v3 of this action (and we could release v3 right away)